src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -81,36 +79,37 @@ public override void OnStartActivity(Activity activity, object payload)
                 return;
             }
 
+            ActivityContext activityContext = default;
```

Comment:
rename to `parentActivityContextOfOriginalActivity` if you don't mind looong names.